docs/manual/en/introduction/How-to-use-node-material.html

```diff
@@ -0,0 +1,815 @@
+		</p>
+
+		<h2>How it works?</h2>
+
+		<p>
+		A `NodeMaterial` consists of `flows` which can have one or more `Nodes`, you can customize a `NodeMaterial` using existing `Nodes` and also create your own.
+		</p>
+
+		<p>
+		All `Nodes` can be connected to each other, and all must have automatic conversion according to the needs of their connections. Each Node and can vary between number of inputs and a single output with dynamic types.
+		`Nodes` can be values such as *Uniforms*, *Constants*, *Attributes of a Geometry* and *Mathematical Functions*.
+		</p>
+
+		<p>
+		`Node` is responsible for emitting an snippet that can vary depending on the input arguments and output expected by the parent `Node`.
+		These small code represents the final result of `flow` of each property of a `NodeMaterial`. This code is generated from the `Node.generate()` function and changes according with the `Context` and `NodeBuilder` used.
+		</p>
+
+		<h2>Creating a simple NodeMaterial</h2>
+
+		<h4>Example</h4>
+		<code>
+import { MeshStandardNodeMaterial, color } from 'three-nodes/Nodes.js';
+
+const material = new MeshStandardNodeMaterial();
+material.colorNode = color( 0xFFFFFF );
+		</code>
+
+		<h1>Node Frame</h1>
+		<p>
+		`NodeFrame` is responsible for updating all `Nodes` per `FRAME` or per `OBJECT`.
+		The main difference here is that some `Nodes` need to be updated throughout the rendering these are updated as `OBJECT`, while others can share input values with multiple `RenderTargets` and others objects, these are updated per `FRAME`.
+		</p>
+
+		<h4>Example</h4>
+		<code>
+import { nodeFrame } from './jsm/renderers/webgl/nodes/WebGLNodes.js';
+
+function animate() {
+
+	// updates the frame state
+	nodeFrame.update();
+
+	renderer.render( scene, camera );
+
+}
+		</code>
+
+		<h1>Shader Node</h1>
+		<p>
+		`NodeMaterial API` offers one own abstraction language so you don't have to rewrite `Shaders` for different outputs.
+		This also makes it easier to optimize the shaders themselves for the resources used in *three.js* since `NodeBuilder` can share the nodes.
+		</p>
+
+		<p>
+		The syntax is a combination of JS + GLSL, you can use JS to create the variables and flow while uses GLSL functions.
+		</p>
+
+		<h2>Constants and explicit conversions</h2>
+
+		<p>
+		Input functions can be used to create code constants and explicit conversions.
+		`NodeMaterial API` does all conversions automatically if the output and input are different.
+		</p>
+
+		<table>
+			<thead>
+				<tr>
+					<th>Function</th>
+					<th>Description</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>
+						[page:bool]( [param:Boolean value] )
+						<br>
+						[page:bool]( value )
+					</td>
+					<td>Returns a constant boolean value or convert a node to boolean.</td>
+				</tr>
+				<tr>
+					<td>
+						[page:color]( [param:Number value] )
+						<br>
+						[page:color]( [param:Number r], [param:Number g], [param:Number b] )
```

Comment:
I think signatures like `bool( Node value )` and `color( Node value, Node value, Node value )` also should be added here.